tests/integration/user_management/claims_repository/test_seed.py

```diff
@@ -18,21 +18,28 @@
 
 import asyncio
 import logging
+from datetime import datetime
+from typing import Any
 
 import pytest
+from ghga_service_commons.utils.utc_dates import now_as_utc
 from hexkit.providers.akafka.testutils import KafkaFixture
 from hexkit.providers.mongodb.testutils import MongoDbFixture
 
 from auth_service.config import Config
 from auth_service.user_management.claims_repository.core.seed import (
     seed_data_steward_claims,
 )
+from auth_service.user_management.claims_repository.models.config import (
+    DataStewardInfo,
+    IvaType,
+)
 
 
-def test_add_non_existing_data_steward(
+def test_add_data_steward(
```

Comment:
Yes, fixed in 7f9fb242. I guess it had been implemented that way because we had an issue with incompatibility of test scopes and associated event loops in the past.